stacks/abax-vwfs/Pulumi.prod.yaml

```diff
@@ -0,0 +1,6 @@
+environment:
+  - abax-vwfs-prod
+config:
+  portal-image: europe-north1-docker.pkg.dev/branches-org-main/branches-org-main/abax-vwfs/portal
+  tag: main-3e4a710
```

Comment:
Suggest that you move this to `abax-vwfs-prod` environment :)

Reply:
Even the tag?